daffodil-core/src/main/scala/org/apache/daffodil/dpath/Expression.scala

```diff
@@ -1634,6 +1634,16 @@ case class FunctionCallExpression(functionQNameString: String, expressions: List
       case (RefQName(_, "rightShift", DFDLX), args) =>
         DFDLXShiftExpr(functionQNameString, functionQName, args,
           DFDLXRightShift(_, _))
+      case (RefQName(_, "xor", DFDLX), args) =>
```

Comment:
xor, and, or should be renamed to xorBits, andBits, and orBits to clearly distinguish them from boolean operations.

XPath already has regular and/or operators. 

Given that these new things are functions, not operators, and they're in the dfdlx namespace that may seem like overkill, but I would propose that these get added to the DFDL expression language officially at some point, at which point the namespace prefix would go away and they might be recast as infix operators. 

Do we have a notBits operator. We need that also.

Reply:
For notBits we will have just one argument?

Reply:
Yes. That means another base class slightly different than the one you have for 2-args bits functions.

Reply:
I have added not.